packages/framework/presence/api-report/presence.alpha.api.md

```diff
@@ -37,8 +37,8 @@ export interface IPresence {
 
 // @alpha @sealed
 export interface ISessionClient<SpecificSessionClientId extends ClientSessionId = ClientSessionId> {
-    connectionId(): ClientConnectionId;
-    getStatus(): SessionClientStatus;
+    getConnectionId(): ClientConnectionId;
+    getConnectionStatus(): SessionClientStatus;
     // (undocumented)
```

Comment:
Docs while we're here? 😁

Reply:
There are docs. (Doesn't say undocumented.)

Reply:
nevermind - UI